app/src/main/res/values/strings.xml

```diff
@@ -93,6 +93,7 @@
     <string name="help">Help</string>
 
     <!-- images to PDF -->
+    <string name="images_to_pdf">Images to PDF</string>
```

Comment:
add this string for other local files also (in english only)

Reply:
Done